app/_includes/head.html

```diff
@@ -49,7 +49,6 @@
   <link rel="dns-prefetch" href="//cdn.segment.com">
   <link rel="dns-prefetch" href="//sidecar.gitter.im">
   <link rel="dns-prefetch" href="//buttons.github.io">
-  ##preload_assets##
```

Comment:
@theetrain I had to remove this because it was showing up in my body - thoughts? It was added with 8ba0ecd626177c10ae15e3414b672f1c5032b7b4